sites/docs/pages/deployment/overview.md

```diff
@@ -8,10 +8,12 @@ description: Evidence is a static site generator, so can be deployed to any stat
 
 # Deployment Overview
 
-In production, Evidence is a [static site generator](https://www.netlify.com/blog/2020/04/14/what-is-a-static-site-generator-and-3-ways-to-find-the-best-one/). This means it doesn't run queries when someone visits your site, but pre-builds all possible pages as HTML beforehand.
+In production, Evidence is a [static site generator](https://www.netlify.com/blog/2020/04/14/what-is-a-static-site-generator-and-3-ways-to-find-the-best-one/) by default. This means it doesn't run queries when someone visits your site, but pre-builds all possible pages as HTML beforehand.
 
 You can host your Evidence project using Evidence Cloud, cloud services like Netlify or Vercel, or your own infrastructure. Evidence does not currently support Github Pages, there is more information on [GitHub](https://github.com/evidence-dev/evidence/issues/603).
 
+You can also configure Evidence as a [Single Page App (SPA)](/deployment/rendering-modes). This means that Evidence will not pre-build all the pages in your application. This can be useful if you have a very large number of pages (c.1000+) causing long build times.
```

Comment:
```suggestion
You can also configure Evidence as a [Single Page App (SPA)](/deployment/rendering-modes). In SPA mode Evidence will not pre-build all the pages in your application. This can be preferrable if your app has many pages (>1,000+) causing long build times.
```